--- a/Makefile
+++ b/Makefile
@@ -267,19 +267,19 @@ ifeq ($(CONFIG_RTL8188E), y)
 
 RTL871X = rtl8188e
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8189es
+MODULE_NAME = rtl8189es
 endif
 
 ifeq ($(CONFIG_GSPI_HCI), y)
-MODULE_NAME = 8189es
+MODULE_NAME = rtl8189es
 endif
 
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8188eu
+MODULE_NAME = rtl8188eu
 endif
 
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8188ee
+MODULE_NAME = rtl8188ee
 endif
 ccflags-y += -DCONFIG_RTL8188E
 
@@ -328,15 +328,15 @@ ifeq ($(CONFIG_RTL8192E), y)
 
 RTL871X = rtl8192e
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8192es
+MODULE_NAME = rtl8192es
 endif
 
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8192eu
+MODULE_NAME = rtl8192eu
 endif
 
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8192ee
+MODULE_NAME = rtl8192ee
 endif
 ccflags-y += -DCONFIG_RTL8192E
 _HAL_INTFS_FILES += hal/HalPwrSeqCmd.o \
@@ -389,13 +389,13 @@ ifneq ($(CONFIG_RTL8812A)_$(CONFIG_RTL88
 
 RTL871X = rtl8812a
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8812au
+MODULE_NAME = rtl8812au
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8812ae
+MODULE_NAME = rtl8812ae
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8812as
+MODULE_NAME = rtl8812as
 endif
 
 _HAL_INTFS_FILES +=  hal/HalPwrSeqCmd.o \
@@ -495,13 +495,13 @@ ifeq ($(CONFIG_RTL8723B), y)
 
 RTL871X = rtl8723b
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8723bu
+MODULE_NAME = rtl8723bu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8723be
+MODULE_NAME = rtl8723be
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8723bs
+MODULE_NAME = rtl8723bs
 endif
 
 ccflags-y += -DCONFIG_RTL8723B
@@ -556,13 +556,13 @@ ifeq ($(CONFIG_RTL8814A), y)
 ##########################################
 RTL871X = rtl8814a
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8814au
+MODULE_NAME = rtl8814au
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8814ae
+MODULE_NAME = rtl8814ae
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8814as
+MODULE_NAME = rtl8814as
 endif
 
 ccflags-y += -DCONFIG_RTL8814A
@@ -614,15 +614,15 @@ ifeq ($(CONFIG_RTL8723C), y)
 
 RTL871X = rtl8703b
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8723cu
+MODULE_NAME = rtl8723cu
 MODULE_SUB_NAME = 8703bu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8723ce
+MODULE_NAME = rtl8723ce
 MODULE_SUB_NAME = 8703be
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8723cs
+MODULE_NAME = rtl8723cs
 MODULE_SUB_NAME = 8703bs
 endif
 
@@ -670,15 +670,15 @@ ifeq ($(CONFIG_RTL8723D), y)
 
 RTL871X = rtl8723d
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8723du
+MODULE_NAME = rtl8723du
 MODULE_SUB_NAME = 8723du
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8723de
+MODULE_NAME = rtl8723de
 MODULE_SUB_NAME = 8723de
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8723ds
+MODULE_NAME = rtl8723ds
 MODULE_SUB_NAME = 8723ds
 endif
 
@@ -729,13 +729,13 @@ ifeq ($(CONFIG_RTL8188F), y)
 
 RTL871X = rtl8188f
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8188fu
+MODULE_NAME = rtl8188fu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8188fe
+MODULE_NAME = rtl8188fe
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8189fs
+MODULE_NAME = rtl8189fs
 endif
 
 ccflags-y += -DCONFIG_RTL8188F
@@ -779,10 +779,10 @@ ifeq ($(CONFIG_RTL8188GTV), y)
 
 RTL871X = rtl8188gtv
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8188gtvu
+MODULE_NAME = rtl8188gtvu
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8189gtvs
+MODULE_NAME = rtl8189gtvs
 endif
 
 ccflags-y += -DCONFIG_RTL8188GTV
@@ -826,16 +826,16 @@ ifeq ($(CONFIG_RTL8822B), y)
 RTL871X := rtl8822b
 ifeq ($(CONFIG_USB_HCI), y)
 ifeq ($(CONFIG_BT_COEXIST), n)
-MODULE_NAME = 8812bu
+MODULE_NAME = rtl8812bu
 else
-MODULE_NAME = 88x2bu
+MODULE_NAME = rtl88x2bu
 endif
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 88x2be
+MODULE_NAME = rtl88x2be
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 88x2bs
+MODULE_NAME = rtl88x2bs
 endif
 
 endif
@@ -843,13 +843,13 @@ endif
 ifeq ($(CONFIG_RTL8821C), y)
 RTL871X := rtl8821c
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8821cu
+MODULE_NAME = rtl8821cu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8821ce
+MODULE_NAME = rtl8821ce
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8821cs
+MODULE_NAME = rtl8821cs
 endif
 
 endif
@@ -859,7 +859,7 @@ ifeq ($(CONFIG_RTL8710B), y)
 
 RTL871X = rtl8710b
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8710bu
+MODULE_NAME = rtl8710bu
 MODULE_SUB_NAME = 8710bu
 endif
 
@@ -898,15 +898,15 @@ ifeq ($(CONFIG_RTL8192F), y)
 
 RTL871X = rtl8192f
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8192fu
+MODULE_NAME = rtl8192fu
 MODULE_SUB_NAME = 8192fu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8192fe
+MODULE_NAME = rtl8192fe
 MODULE_SUB_NAME = 8192fe
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 8192fs
+MODULE_NAME = rtl8192fs
 MODULE_SUB_NAME = 8192fs
 endif
 
@@ -962,16 +962,16 @@ ifeq ($(CONFIG_RTL8822C), y)
 RTL871X := rtl8822c
 ifeq ($(CONFIG_USB_HCI), y)
 ifeq ($(CONFIG_BT_COEXIST), n)
-MODULE_NAME = 8812cu
+MODULE_NAME = rtl8812cu
 else
-MODULE_NAME = 88x2cu
+MODULE_NAME = rtl88x2cu
 endif
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 88x2ce
+MODULE_NAME = rtl88x2ce
 endif
 ifeq ($(CONFIG_SDIO_HCI), y)
-MODULE_NAME = 88x2cs
+MODULE_NAME = rtl88x2cs
 endif
 
 endif
@@ -980,10 +980,10 @@ endif
 ifeq ($(CONFIG_RTL8814B), y)
 RTL871X := rtl8814b
 ifeq ($(CONFIG_USB_HCI), y)
-MODULE_NAME = 8814bu
+MODULE_NAME = rtl8814bu
 endif
 ifeq ($(CONFIG_PCI_HCI), y)
-MODULE_NAME = 8814be
+MODULE_NAME = rtl8814be
 endif
 
 endif
